### ISSUE TITLE
Mismatch qt path

### ISSUE THREAD

**Issue by vieelblond** [therecipe/qt#581]
When is defined  custom path for QT, is setted wrong path for QT binaries in some compilations (not for everythings)

For example this config give you  /store/qt/5.10.1/5.10.1 
```bash
export QT_VERSION=5.10.1
export QT_DIR=/store/qt
export QT_STUB=false
export QT_DEBUG=false
```

```log
ERRO[0022] failed to run command                         cmd="/store/qt/5.10.1/5.10.1/clang_64/bin/rcc -project -o /Volumes/xxxx/workspace/gocode/src/github.com/therecipe/qt/internal/examples/qml/application/rcc.qrc" dir=/Volumes/xxxx/workspace/gocode/src/github.com/therecipe/qt/internal/examples/qml/application/qml env= error="fork/exec /store/qt/5.10.1/5.10.1/clang_64/bin/rcc: no such file or directory" func=RunCmd name="execute rcc *.qrc on darwin for darwin"
```

**Comment by therecipe** [comment]
Hey

Seems like you set `QT_DIR` wrong.
`QT_DIR` should point to the root Qt installation folder.

Maybe check your env again.
If `QT_DIR` contains the `5.10.1` folder, then there is also no need to explicitly export `QT_VERSION`.

**Comment by creedr** [comment]
Hello,
I spent a while today trying to get therecipe/qt working with my installs of Qt and Go on my new Windows 10 machine, and ran into this issue. 

I had first set my `QT_DIR` to `E:\Qt\5.10.1` similar to how the [instructions](https://github.com/therecipe/qt/wiki/Installation) noted to set as `C:\Qt\5.10.0`, and I got an error like @maregis mentioned. I fixed that by changing my `QT_DIR` to the root Qt folder and removing `QT_VERSION`. Perhaps this could have been stated more clearly in the installation [instructions](https://github.com/therecipe/qt/wiki/Installation) under **Regular Installation.**

Once I got that error fixed, I ran into another error (I don't remember exactly what it was now) that the setup was looking for a Qt version `5.10.0`. So I went back and installed `Qt 5.10.0` and made that error dissappear, so maybe it's just me, but perhaps this package just doesn't work with `5.10.1` on Windoes yet?

Anyways, with `5.10.0` installed, the `qtsetup` ran and the tests appeared to work right, although there were a few other ignored errors because I didn't have everything installed e.g. the Android stuff. 

Then I tried the [Widget example here](https://github.com/therecipe/qt/wiki/Getting-Started#starting-application) and got an error, something along the lines of, the MinGW 32 bit can't be used to compile this program (that was the MinGW 32 bit installed by the Qt Installer, and they don't have a 64 bit version.) I had some other issues there that I was able to fix with a change to some code in `therecipe\qt` package, but I should write those up in a separate issue.
